config/config.default.php

```diff
@@ -266,9 +266,13 @@
     // Whether the mobile number can be shown to other users
     'enable_mobile_show'      => (bool) env('ENABLE_MOBILE_SHOW', false),
 
-    // Enables prename and lastname
+    // Enables firstname and lastname
     'enable_user_name'        => (bool) env('ENABLE_USER_NAME', false),
 
+    // Show a users firstname and lastname instead of username
+    'display_full_user_name'  => env('DISPLAY_FULL_USER_NAME', false)
```

Comment:
The description says „instead of user name“ but the option is `display_full_user_name`.

For less confusion I would name this `display_full_name`.